src/store/store.js

```diff
@@ -0,0 +1,35 @@
+import Vuex from 'vuex'
+import Vue from 'vue'
+Vue.use(Vuex)
+const state = {
+    isAuthenticated: false,
+    user: null
+}
+const mutations = {
+    SET_AUTHENTICATED(state, isAuthenticated) {
```

Comment:
naming should always be consistent everywhere in the whole application. Since in most of the places you used cameCase for functions and validables. Make the same for mutations as well. setAuthenticated.